dockerfile

```diff
@@ -0,0 +1,45 @@
+# Use the official Ubuntu 22.04 as a base image
+FROM ubuntu:22.04 AS base
```

Comment:
For any new images, could we try starting with `24.04`? We should eventually update all our images like so.

Reply:
Updated to `24.04`